### PR TITLE
Premium plan block

### PR DESCRIPTION
**Pull request by williambsm**
This is the first draft of the premium plan block.

Mobile XD:
https://xd.adobe.com/view/91c324fe-4415-48a3-a0f6-86a204aceba4-c490/
Desktop XD:
https://xd.adobe.com/view/c4c1a521-4ed4-42c3-aec1-30ab2d1d65cc-9192/specs/
Test URL:
https://premium-plan-block--express-website--webistry-development.hlx.page/drafts/williambsm/pricing-new?lighthouse=on

### CODE REVIEW

File: express/blocks/premium-plan/premium-plan.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+      if ($cardLink) {
+        $card = createTag('a', { class: 'premium-plan-card', href: $cardLink.href });
+        $cardLink.remove();
+        $card.innerHTML = $cardDiv.innerHTML;
+      } else {
+        $card = $cardDiv;
+        $card.classList.add('premium-plan-card');
+      }
+
+      if (device === 'Mobile') {
```
Comment (rofe) [inline express/blocks/premium-plan/premium-plan.js:81]: I would probably `toLowerCase()` the author input to be on the safe side

Comment (rofe) [reply express/blocks/premium-plan/premium-plan.js:81]: And `trim()` 😉

File: express/blocks/shared/carousel.js [hunk 2]
```diff
@@ -63,10 +63,10 @@ export function buildCarousel(selector = ':scope > *', $parent, infinityScrollEn
     toggleControls();
   };
   $arrowLeft.addEventListener('click', () => {
-    moveCarousel(240);
+    moveCarousel($platform.offsetWidth);
```
Comment (rofe) [inline express/blocks/shared/carousel.js:66]: Caution: @So1ene is modifying the same code in #437 but slightly differently. You may want to align here...

**Review by rofe — CHANGES_REQUESTED** [review]

File: express/blocks/shared/carousel.js (hunk 2 above)
Comment (So1ene) [reply express/blocks/shared/carousel.js:66]: I did this, I prefer the solution in https://github.com/adobe/express-website/pull/437

**Review by rofe — APPROVED** [review]

**Pull request merged**